ui/components/editor-channel-dialog.vue

```diff
@@ -359,6 +364,55 @@ export default {
       this.resetChannelForm();
     },
 
+    onChannelNameChanged(name) {
```

Comment:
Better rename the parameter `name` to `channelName` to avoid confusion in the function.